indexes/plugin.js

```diff
@@ -84,8 +84,8 @@ module.exports = class Plugin {
         if (record.value) this.processRecord(record, processedSeq)
```

Comment:
The `if` on line 83: I understand that `this.offset.value` means "the `offset` that is currently persisted in the META section on leveldb". Would it be safer to use `record.offset > processedOffset`? Notice that `processedOffset` is also loaded from the leveldb, but is guaranteed to be a bit more ahead of `this.offset.value`.

Reply:
Correct, `processedOffset` can be ahead. It should not be a problem since we only have 1 place calling this function: https://github.com/ssb-ngi-pointer/ssb-db2/blob/86d502f0846c3c234f5f6ff47be3f5e4bbe58c20/db.js#L502. Hmm, I'm a bit torn here if we fix that or not.

Reply:
I think you are right, lets fix this properly while we are at it. Before if for some reason the same offset was sent there would be a bug where seq would be incorrect. I've pushed another commit.